deepchem/feat/molecule_featurizers/mat_featurizer.py

```diff
@@ -2,6 +2,29 @@
 from deepchem.utils.molecule_feature_utils import one_hot_encode
 from deepchem.utils.typing import RDKitMol, RDKitAtom
 import numpy as np
+from typing import Tuple, Any
+from dataclasses import dataclass
+
+
+@dataclass
```

Comment:
Could you tell me a bit more about the dataclass annotation? We haven't used this before in DeepChem.